meta/parse.pl

```diff
@@ -72,6 +72,8 @@
 my $FLAGS = "MANDATORY_ON_CREATE|CREATE_ONLY|CREATE_AND_SET|READ_ONLY|KEY";
 my $ENUM_FLAGS_TYPES = "(none|strict|mixed|ranges|free)";
 
+my $DECIMAL_P2_REGEX = '([+-]?([0-9]*[.])?[0-9]{0,2})';
```

Comment:
double can be written in expotential form, as you are introducing so many changes instead of using int/uint

Reply:
this is how yang type defined. not me.

Reply:
that's not an argument to force this specific defintion here if your generator don't generate that